ApplyToBecomeInternal/ApplyToBecomeInternal/Models/ProjectViewModel.cs

```diff
@@ -34,5 +49,14 @@ public ProjectViewModel(Project project)
 		public string ApplicationReceivedDate { get; }
 		public string AssignedDate { get; }
 		public string Phase { get; }
+		public string Section
```

Comment:
This is logic that belongs in a view model for the SubMenu as it's not relevant to a Project.